crystal_diffusion/data/parse_lammps.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I think this script could go in [TOP_DIRECTORY[]/data since it analyses a specific dataset.

I don't think this script can run right now, since nothing in the lammps script writes `thermo_log.yaml`? Is there another file that needs to be commited?